qa/L0_cuda_shared_memory/cuda_shared_memory_test.py

```diff
@@ -331,7 +331,7 @@ def test_register_out_of_bound(self):
         # Verify various edge cases of registered region size don't go out of bounds of the actual created shm region's size.
         with self.assertRaisesRegex(
             InferenceServerException,
-            r"failed to register CUDA shared memory region.*register size exceeds CUDA shared memory region size",
+            "failed to register shared memory region.*invalid args",
```

Comment:
failed to register `CUDA` shared memory region

Reply:
Yes, the api for cuda shm is `cuda_shared_memory` and for system shm is `register_system_shared_memory`, so the user should know if this is for cuda shm or system shm. In this case I think we could provide the `CUDA` keyword in the error message if we want.

Reply:
I updated the error message to include `CUDA` keyword for now. @nnshah1 Let me know if there's any concern for this.